conference.js

```diff
@@ -1612,31 +1605,66 @@ export default {
      * @return {Promise<JitsiStreamPresenterEffect>} - A promise resolved with
      * {@link JitsiStreamPresenterEffect} if it succeeds.
      */
-    async _createPresenterStreamEffect(height, cameraDeviceId = null) {
-        let presenterTrack;
+    async _createPresenterStreamEffect(height = null, cameraDeviceId = null) {
+        if (!this.localPresenterVideo) {
+            let presenterTrack;
```

Comment:
why the auxiliary variable? you could assign it to `this.localPresenterVideo` directly

Reply:
Done